src/src/rx_main.cpp

```diff
@@ -14,7 +14,13 @@
 #include "options.h"
 #include "MeanAccumulator.h"
 
-#include "devCRSF.h"
+#include "rx-serial/SerialIO.h"
+#include "rx-serial/SerialNOOP.h"
+#include "rx-serial/SerialCRSF.h"
+#include "rx-serial/SerialSBUS.h"
+#include "rx-serial/SerialAirPort.h"
```

Comment:
Oh nice!

Reply:
We should be able to add more protocols easily now too.